src/CONST.ts

```diff
@@ -46,6 +46,8 @@ const CONST = {
         IN: 'in',
         OUT: 'out',
     },
+    // Multiplier for gyroscope animation in order to make it a bit more subtle
+    ANIMATION_GYROSCOPE_VALUE: 0.65,
```

Comment:
What was the previous value? Just 1?

Reply:
Yes, it's a multiplier value I have just added, therefore previously it was 1 😄